sats-dna/src/main/kotlin/com/sats/dna/components/card/SatsArticleCard.kt

```diff
@@ -1,45 +1,68 @@
-package com.sats.dna.components
+package com.sats.dna.components.card
 
 import androidx.compose.foundation.clickable
 import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.aspectRatio
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.platform.LocalInspectionMode
 import androidx.compose.ui.tooling.preview.PreviewLightDark
 import coil.compose.AsyncImage
 import com.sats.dna.LocalUseMaterial3
-import com.sats.dna.components.card.SatsCard
+import com.sats.dna.components.SatsTag
+import com.sats.dna.components.SatsTagColor
+import com.sats.dna.components.button.SatsDismissButton
 import com.sats.dna.theme.SatsTheme
 import com.sats.dna.tooling.PreviewImagePlaceholder
 
 @Composable
-fun SatsCampaignModule(
+fun SatsArticleCard(
     imageUrl: String,
     title: String,
     subtitle: String?,
     onClick: () -> Unit,
     modifier: Modifier = Modifier,
+    tag: @Composable (() -> Unit?)? = null,
+    dismissButton: @Composable (() -> Unit?)? = null,
 ) {
     CompositionLocalProvider(LocalUseMaterial3 provides true) {
         SatsCard(modifier) {
             Column(Modifier.clickable { onClick() }) {
-                HeroImage(imageUrl)
+                Box {
+                    HeroImage(imageUrl)
+                    Row(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(start = SatsTheme.spacing.s),
+                        horizontalArrangement = Arrangement.SpaceBetween,
+                        verticalAlignment = Alignment.CenterVertically,
+                    ) {
+                        tag?.let {
+                            tag()
+                        }
+                        dismissButton?.let {
+                            dismissButton()
+                        }
+                    }
```

Comment:
Will this look correct if it has a dismiss button but no tag? If I'm reading this correctly, the dismiss button be top left in that case? 🤔